tests/Integration/Resources/Storage/Test_Resources_GFAL2StorageBase.py

```diff
@@ -1,38 +1,63 @@
 """
-This tests the new plugins for rel-v6r13: GFAL2_SRM2Storage and GFAL2_XROOTStorage.
-
-Plugins:
-
-To run the test following plugins are needed:
-
-gfal2-2.9.3
-gfal2-plugin-xrootd-0.4.0
-gfal2-python-1.8.3
-
-Storage element:
-Change self.storageName within the basicTest class to the name of the storageElement you wish to run this test against.
-This storage element needs to be supporting two access protocols, using the protocolName GFAL2_SRM2 and GFAL2_XROOT
-respectively. If you want to test only either SRM2 or XROOT remove the other testsuite in the unittest.
-
+This integration tests will perform basic operations on a storage element, depending on which protocols are available.
+It creates a local hierarchy, and then tries to upload, download, remove, get metadata etc
```

Comment:
Can you provide an example on how to run this test?

Also, I guess it needs a "real" SE behind. So, a "real" certificate, correct?